Application/Api/Education/Certificate/Generator/Repository/ESBD/EsbdGymKurshalbjahreszeugnis.php

```diff
@@ -0,0 +1,679 @@
+                    }
+
+                    if (($tblCategory = Subject::useService()->getCategoryByIdentifier('FOREIGNLANGUAGE'))
+                        && Subject::useService()->existsCategorySubject($tblCategory, $tblSubject)
+                    ) {
+                        $isLanguage = true;
+                    } else {
+                        $isLanguage = false;
+                    }
+
+                    $this->setCourseSubject($tblSubject, $section, false, $IsGradeUnderlined, $personId,
+                        $isShrinkMarginTop ? '2px' : '10px', false, !$isAdvancedCourse, $isLanguage);
+
+                    if ($isLanguage) {
+                        $subSection = new Section();
+                        $subSection
+                            ->addElementColumn((new Element())
+                                ->setContent('Fremdsprache')
+                                ->styleMarginTop('0px')
+                                ->styleMarginBottom('0px')
+                                ->styleTextSize('8px')
+                            );
+                    }
+
+                    if ($isSecondLane) {
+                        $slice->addSection($section);
+                        if ($subSection) {
+                            $slice->addSection($subSection);
+                            $subSection = false;
+                            $isShrinkMarginTop = true;
+                        } else {
+                            $isShrinkMarginTop = false;
+                        }
+                        $section = new Section();
+                    } else {
+                        $section
+                            ->addElementColumn((new Element())
+                                ->setContent('&nbsp;')
+                                ->styleMarginTop($isShrinkMarginTop ? '2px' : '10px')
+                                , '8%');
+                    }
+
+                    $isSecondLane = !$isSecondLane;
+                }
+            }
+
+            if ($isSecondLane) {
+                $section
+                    ->addElementColumn((new Element())
+                        ->setContent('&nbsp;')
+                    );
+                $slice->addSection($section);
+            }
+        }
+
+        return $slice
+            ->styleHeight('290px');
+    }
+
+    /**
+     * @param TblSubject|null $tblSubject
+     * @param Section $section
+     * @param $isSubjectUnderlined
+     * @param $isGradeUnderlined
+     * @param $personId
+     * @param string $marginTop
+     * @param bool $isFootnote
+     * @param bool $isGradeShown
+     * @param bool $isLanguage
+     */
+    private function setCourseSubject(
+        TblSubject $tblSubject = null,
+        Section $section,
+        $isSubjectUnderlined,
+        $isGradeUnderlined,
+        $personId,
+        $marginTop = '10px',
+        $isFootnote = false,
+        $isGradeShown = true,
+        $isLanguage = false
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Incorrect spacing between argument "$isLanguage" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5646354)